packages/map-template/src/components/LocationDetails/LocationDetails.scss

```diff
@@ -67,7 +69,8 @@
             color: var(--color-gray-70);
             line-height: 1.5;
             overflow-y: hidden;
-            max-height: var(--spacing-xxxx-large); /* corresponding to approx. 3 lines of description text */
+            max-height: var(--spacing-xxxx-large);
+            /* corresponding to approx. 3 lines of description text */
```

Comment:
Move comment up again (and tell your editor not to autofix this 😄 )

Reply:
It's very insistent on it 😄